app/code/core/Mage/Adminhtml/Helper/Help/Mapping.php

```diff
@@ -231,7 +231,7 @@ protected function findInMapping($frontModule, $controllerName, $actionName)
     /**
      * Determine which version of docs should target onto
      *
-     * @return string
+     * @return false|Varien_Simplexml_Element
```

Comment:
```suggestion
     * @return Mage_Core_Model_Config_Element|Varien_Simplexml_Element|false
```

Reply:
Thought about it ....

I thinks with strict types on phpstan needs a class that implements `__toString()`.

`Mage_Core_Model_Config_Element` extends `Varien_Simplexml_Element` so i used this.

To narrow it down, how about `Mage_Core_Model_Config_Element|false`?

(but yes, add all types from parent method is also okay)